.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
```

Comment:
Hmm , this file shouldn't be changed; backmerge should fix. 2.2 might need a backmerge from dev too, but unlikely